packages/components/package.json

```diff
@@ -44,6 +44,7 @@
     "url": "https://github.com/Talend/ui.git"
   },
   "dependencies": {
+    "date-fns": "1.27.2",
```

Comment:
if we start to do that, we should use `^` inside deps and no `^`
So the project is still able to control versions of the dependencies.

Reply:
I don't understand your point?

Reply:
I don't understand your first sentence. But I think I get the idea. You said that we should add a `^` wildcard to avoid apps to get 2 versions  if they rely on a version that can fit `^1.27.2` right ?

Reply:
yes. this need to re-work the version.js

Reply:
I'll change it and merge this PR